packages/uswds/src/Templates/WrapIfAdditionalTemplate.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Grid, Label, TextInput } from '@trussworks/react-uswds';
+import {
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+  WrapIfAdditionalTemplateProps,
+  TranslatableString,
+} from '@rjsf/utils';
+
+export default function WrapIfAdditionalTemplate<
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>(props: WrapIfAdditionalTemplateProps<T, S, F>) {
+  const {
+    children,
+    disabled,
+    id,
+    label,
+    onDropPropertyClick,
+    onKeyChange,
+    readonly,
+    required,
+    schema,
+    uiSchema,
+    registry,
+  } = props;
+  const { templates, translateString } = registry;
+  const keyLabel = translateString(TranslatableString.KeyLabel, [label]);
+  const additional = schema.additionalProperties;
+
+  if (!additional) {
+    return <>{children}</>;
+  }
```

Comment:
You've lost the marker classNames

```suggestion
  if (!additional) {
    return <div className={classNames}>{children}</div>;
  }
```